handbook/communication/content_guidelines/index.md

```diff
@@ -0,0 +1,33 @@
+# Sourcegraph content guidelines
+
+The goal of these content guidelines is to help us to:
+
+- **Create consistency** across our product, marketing material, documentation, and handbook.
+- **Empower our team** to make easier for anyone to write clear, effective, and useful content.
+- **Drive positive change** in language standards to create compassionate, inclusive, and respectful products.
+
+### [Voice and tone](./voice_and_tone.md)
+### [Actionable language](./actionable_language.md)
+### [Style & mechanics](./style_and_mechanics.md)
+### [Word list](./word_list.md)
+
+## Effectiveness over correctness
+
+These content guidelines aren't about "correct" and "incorrect" writing. It is about effective writing for our target users. Of course, effectiveness involves correctness to some degree, but only correctness as judged by the audience, not as judged by an appeal to authority.
+
+The examples included in these guidelines are for comparison's sake, not for exactness.
+
+These guidelines are a living reference, and we welcome changes. The most effective arguments for changes are those based on effectiveness, not correctness.
+
+For further reading, see [linguistic prescription on Wikipedia](https://en.wikipedia.org/wiki/Linguistic_prescription).
+
```

Comment:
```suggestion

## Effectiveness over correctness

These content guidelines aren't about "correct" and "incorrect" writing. It is about effective writing for our target users. Of course, effectiveness involves correctness to some degree, but only correctness as judged by the audience, not as judged by an appeal to authority.

The examples included in these guidelines are for comparison's sake, not for exactness.

These guidelines are a living reference, and we welcome changes. The most effective arguments for changes are those based on effectiveness, not correctness.

For further reading, see [linguistic prescription on Wikipedia](https://en.wikipedia.org/wiki/Linguistic_prescription).

### [Voice and tone](./voice_and_tone.md)
### [Actionable language](./actionable_language.md)
### [Style & mechanics](./style_and_mechanics.md)
### [Word list](./word_list.md)

```
I think the **Effectiveness over correctness** section would be better before the page list, because it serves as an additional intro to what we're trying to accomplish with the guidelines